Build/Tactical/TeamTurns.cc

```diff
@@ -202,6 +202,11 @@ void EndTurn( UINT8 ubNextTeam )
 		FOR_EACH_IN_TEAM(s, gTacticalStatus.ubCurrentTeam)
 		{
 			s->bMoved = TRUE;
+			// Cancel merc movement if continue path was not used
+			if ( CheckForMercContMove(s)) {
```

Comment:
I prefer the terser variant, but we should just follow whatever is there already or it will get messy.